src/main/java/HumanPlayer.java

```diff
@@ -0,0 +1,30 @@
+import java.util.List;
+
+public class HumanPlayer implements Player{
+
+    private String name = "";
+    private String marker = "";
+
+    public String getName() {
+        return name;
+    }
+
+    public String getMarker() {
+        return marker;
+    }
+
+    public void build(String name, String marker) {
```

Comment:
Why not using the class constructor?

```
public class HumanPlayer implements Player {
    public HumanPlayer(String name, String marker) {
        this.name = name;
        this.marker = marker;
    }
}
```
then you can set those informations when you create the class

```
HumanPlayer human = new HumanPlayer("your name", "X");
```